shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/core/data/config/FirebaseRemoteConfigDataSource.kt

```diff
@@ -0,0 +1,21 @@
+package com.vml.tutorial.plantshop.core.data.config
+
+import dev.gitlive.firebase.remoteconfig.FirebaseRemoteConfig
+
+interface FirebaseRemoteConfigDataSource {
+    suspend fun getConfig(key: String): String?
+}
+
+class FirebaseRemoteConfigDataSourceImpl(private val firebaseRemoteConfig: FirebaseRemoteConfig) :
+    FirebaseRemoteConfigDataSource {
+    override suspend fun getConfig(key: String): String? {
+        return try {
+            firebaseRemoteConfig.fetchAndActivate()
+            firebaseRemoteConfig.getValue(key).asString()
+
+            firebaseRemoteConfig.
```

Comment:
Is it a leftover line or did you forget to finish it ? :)

Reply:
That was odd :D It is a leftover. Removed